sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -14,4 +14,7 @@ export class ApplicationService {
   public async getPIRSummary(locationId: number): Promise<PIRSummaryDTO[]> {
     return ApiClient.Application.getPIRSummary(locationId);
   }
+  public async confirmationOfAssessment(applicationId: number): Promise<any> {
```

Comment:
IMO this method should not return any data. The success or error could be expressed as HTTP codes.